pkg/controller/installation/installation_controller.go

```diff
@@ -664,6 +664,12 @@ func (r *ReconcileInstallation) preflightChecks(installation *integreatlyv1alpha
 			_, err := strconv.ParseBool(s)
 			return err
 		}),
+		integreatlyv1alpha1.EnvKeyAlertSMTPFrom: requiredEnvVar(func(s string) error {
```

Comment:
@Boomatang just more of a question than anything else, the requiredEnvVar returns an error also, and I wonder would it make sense to return the error from the requiredEnvVar instead, to me, and I might be missing something, we are creating a new error message here with just the addiotion of which envvar is missing - maybe editing the requiredEnvVar so that it would print out also which env var is missing, so that in future, the `checkEnvVars` function could be cleaner with more structured? way of error logging?

Reply:
also, I think we have agreed recently that we could be making these slight improvement to the code as we go, on lines @174 - about 180 we are also checking for envvars, wonder if it would make sense to improve these as well to use the checkEnvVar function? WDYT?

Reply:
Because we are close to release cut date, and this PR is critical item for the release, I would rather put code improvements into a separate PR.

Reply:
yes, I agree with @MStokluska in general, but in this case we should move that to a separate PR - or a Jira.

Reply:
Ok I will create a follow Jira for this